glib/plan.sh

```diff
@@ -14,7 +14,7 @@ pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
 pkg_upstream_url="https://developer.gnome.org/glib/"
 pkg_shasum="82ee94bf4c01459b6b00cb9db0545c2237921e3060c0b74cff13fbc020cfd999"
 pkg_deps=(
-  core/coreutils
```

Comment:
I think we need to keep this as a runtime dependency or we have this build problem:

```
   glib: WARN Could not resolve the path for core/coreutils, is it specified in 'pkg_deps'?
   glib: WARN fix_interpreter() 'core/coreutils' is not a runtime dependency
   glib: WARN Only runtime packages may be used as your interpreter must travel
   glib: WARN with the 'glib' in order to run.
```

Reply:
This is fixed, thanks!

Reply:
Cool.  Don't feel the need to spend /too/ long getting the tests to pass.  If just a few are failing, on thing we've done in other plans is to document what is failing and then future contributors can fix them up.